packages/textlint-tester/src/textlint-tester.ts

```diff
@@ -40,20 +40,32 @@ function assertHasFixer(ruleCreator: any, ruleName: string): any {
     throw new Error(`Not found \`fixer\` function in the ruleCreator: ${ruleName}`);
 }
 
+export type TestConfig = {
+    plugins: { pluginId: string; plugin: TextlintPluginCreator }[];
```

Comment:
I have to add `pluginOptions` and `ruleObject` here, in order to configure each plugin/rule respectively.